config/settings.yml

```diff
@@ -152,11 +152,6 @@
     :addition:
       :critical:
       - CloneTaskEvent
```

Comment:
I'm not sure that this is a valid event, it looks like it is from VMware but it isn't in any of the vmware wsdl files (was added by 7df8572f592033b44ecb4db963f7ea56d85a48e8)
@Fryguy ?